x-pack/plugins/ml/public/settings/calendar/_calendar.scss

```diff
@@ -0,0 +1,15 @@
+.ml-calendar-lists {
+  background: $euiColorLightestShade;
+  min-height: 100vh;
+}
+
+.ml-list-calendar {
+
+  .ml-list-calendar-content {
+    max-width: 1100px;
```

Comment:
You're using this number in multiple files so you should use a variable. Likely it can be replaced with one of our breakpoint values though.

`max-width: map-get($euiBreakpoints, 'xl');`

would get your 1200px

ref: https://github.com/elastic/eui/blob/master/src/global_styling/variables/_responsive.scss

Reply:
Good call - thank you! 😄